README.md

```diff
@@ -42,6 +42,11 @@ For more information, refer to the [documentation](https://docs.unirep.io/)
     | [`utils`](./packages/utils) | <a href="https://www.npmjs.com/package/@unirep/utils"><img alt="NPM version" src="https://img.shields.io/npm/v/@unirep/utils?color=%230004&style=flat-square" /></a> | Utils which are used in unirep protocol. |
     | [`subgraph`](./packages/subgraph/) | - | Subgraph of unirep smart contract. |
 ​
+## 🔋 Requirements
+
+- Intall [rust](https://www.rust-lang.org/tools/install) and [circom 2](https://docs.circom.io/getting-started/installation/)
```

Comment:
This dependency is only for people changing the circuits. I think application developers should be able to use unirep without circom.

Reply:
This is not for application developers. This is for developers who want to contribute to the repo and run the repo locally.
The one for application developers is the readme file in each package and the documentation.

Reply:
> This is not for application developers. This is for developers who want to contribute to the repo and run the repo locally.

I sorta disagree here. A lot of the docs are based around giving application developers an API reference to use when writing unirep applications. The readme's aren't comprehensive enough to help write an application (without having to look at the code).

Why not add a not here saying that rust is only needed for compiling circuits?

Reply:
Oh wait this is the readme for the root repo 🤦, these requirements are fine